data_model/src/lib.rs

```diff
@@ -560,8 +567,8 @@ pub type ValueBox = Box<Value>;
 
 model! {
     /// Sized container for all possible values.
-    #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, FromVariant, EnumDiscriminants, Decode, Encode, PartiallyTaggedDeserialize, PartiallyTaggedSerialize, IntoSchema)]
-    #[strum_discriminants(name(ValueKind), derive(Display, Decode, Encode, Deserialize, Serialize, IntoSchema), cfg_attr(any(feature = "ffi_import", feature = "ffi_export"), derive(iroha_ffi::FfiType)), allow(missing_docs), repr(u8))]
+    #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Decode, Encode, FromVariant, EnumDiscriminants, PartiallyTaggedSerialize, PartiallyTaggedDeserialize, IntoSchema)]
+    #[strum_discriminants(name(ValueKind), derive(Display, Decode, Encode, Serialize, Deserialize, IntoSchema), cfg_attr(any(feature = "ffi_import", feature = "ffi_export"), derive(iroha_ffi::FfiType)), allow(missing_docs), allow(missing_docs), repr(u8))]
```

Comment:
```suggestion
    #[strum_discriminants(name(ValueKind), derive(Display, Decode, Encode, Serialize, Deserialize, IntoSchema), cfg_attr(any(feature = "ffi_import", feature = "ffi_export"), derive(iroha_ffi::FfiType)), allow(missing_docs), repr(u8))]
```

Reply:
Doesn't work because we have no docs for `Value` variants

Reply:
but you have duplication of attributes

Reply:
I think you're missing the point, you have `missing_docs` 3 times. I believe one is redundant. Am I wrong?

Reply:
Oh, yes, I just outlooked that in this diff